### ISSUE TITLE
Engaging

### ISSUE THREAD

**Issue by Pabloso8** [mdbesten/what-could-go-wrong-llm#12]
It is fun the more people you play with.

**Comment by Pabloso8** [comment]
The game by itself allow more players.

**Comment by davimarim** [comment]
That's true! But some alternatives to allow more players to participate are:

1. Playing with more than one Czar (preferably in an odd number to facilitate the process of choosing the discussion card)
2. Create new response cards. The current number of cards (100 cards) may limit the number of players.

I gave some ideas for the second point in my fork (pull request). Have a look and let me know what you think.

**Comment by Pabloso8** [comment]
Really good idea Davi. I may consider these changes on my new branch!

**Comment by ElioBouJaoudeh** [comment]
I like your propositions:)
